wiki/osu!_Program_Files/en.md

```diff
@@ -42,11 +40,11 @@ If you want to know how to do this, see [osu! File Formats](/wiki/osu!_File_Form
 
 ### Localisation
 
-This folder appears when you had switched your Language under Graphics in the Options. **It will hold the translated text files which can be used each to replace the usual English text based on the user's selected localisation.**
+This folder appears when you have switched your language under Graphics in the Options. **It will hold the translated text files which can be used each to replace the usual English text based on the user's selected localisation.**
```

Comment:
```suggestion
This folder appears when you have switched your language in the options. **It will hold the translated text files which can be used each to replace the usual English text based on the user's selected localisation.**
```